extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -408,17 +390,43 @@ require(['wikia.window', 'jquery', 'wikia.nirvana', 'wikia.tracker', 'JSMessages
 							sections: data
 						}
 					}).done(function (data) {
-						if (data.error) {
-							var $items = $form.find('.item-input'),
-								$sections = $form.find('.section-input');
+						var $itemsFeatured = $ul.find('.featured').nextUntil('.section'),
+							$items = $ul.find('.section:not(.featured)').first().nextAll(),
+							$sections = $ul.find('.section:not(.featured)');
 
-							[].forEach.call(data.error, function(err) {
-								var errTitle = err.title,
-									errReason = err.reason,
+						if (data.error) {
+							[].forEach.call(data.error, function(error) {
+								// error := { target: <label>, type: [ item,section,featured], reason: error }
+								var errLabel = error.target,
+									errType = error.type,
+									errReason = error.reason,
 									reasonMessage = gerErrorMessageFromErrReason(errReason);
 
-								checkItemsForErrors($items, errTitle, errReason, reasonMessage);
-								checkSectionsForErrors($sections, errTitle, errReason, reasonMessage);
+								if (errType === 'item') {
```

Comment:
What do you think about converting `if...else` to functions to increase readability (see indentation)